code/src/App.js

```diff
@@ -1,9 +1,68 @@
 import React from 'react'
+import { LIST_URL } from 'utils/urls'
+import { BrowserRouter, Switch, Route } from 'react-router-dom'
+import { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import Header from 'components/Header'
+import MovieList from './components/MovieList'
+import MovieDetails from './components/MovieDetails'
+import LoadingSpinner from 'components/LoadingSpinner'
+import Footer from 'components/Footer'
+import ErrorMessage from 'components/ErrorMessage'
 
 export const App = () => {
+  const [movieTitle, setMovieTitle] = useState([])
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
+    fetchMovieTitle()
+  }, [])
+
+  const fetchMovieTitle = () => {
+    setLoading(true)
+    fetch(LIST_URL)
+      .then((res) => {
+        if (res.ok) {
+          return res.json()
+        } else {
+          throw new Error('something went wrong!')
+        }
+      })
+      .then((data) => setTimeout(() => setMovieTitle(data.results), 1000))
+      .finally(() => setTimeout(() => setLoading(false), 1000))
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <div className="movie-container">
+      {loading && <LoadingSpinner />}
+      <BrowserRouter>
+        <Header />
+        <Switch>
+          <Route exact path='/'>
+            {movieTitle.map((movie) => (
```

Comment:
Is there a reason this is not in the movie list component? I don't remember if you should have it in the component or outside but having it the components makes the app more clean and easy to follow along